my-react-app/src/redux/login/actions.js

```diff
@@ -0,0 +1,31 @@
+import loginService from '../../services/loginService';
+
+export const actions = {
+  GET_TOKEN: 'GET_TOKEN',
+  GET_TOKEN_SUCCESS: 'GET_TOKEN_SUCCESS',
+  GET_TOKEN_FAILURE: 'GET_TOKEN_FAILURE'
+};
+
+const actionCreators = {
```

Comment:
You are not dispatching this actions anywhere.

Reply:
I have dispatched them on screens/Login/index.js on the latest commit